download/src/main/scala/uk/gov/nationalarchives/tdr/export/api/ApiClient.scala

```diff
@@ -0,0 +1,45 @@
+package uk.gov.nationalarchives.tdr.export.api
+
+import java.io.ByteArrayInputStream
+
+import ca.ryangreen.apigateway.generic.{GenericApiGatewayClientBuilder, GenericApiGatewayRequestBuilder, GenericApiGatewayResponse}
+import com.amazonaws.ClientConfiguration
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+import com.amazonaws.http.HttpMethodName
+import com.amazonaws.regions.{Region, Regions}
+import io.circe.generic.auto._
+import io.circe.syntax._
+
+import scala.jdk.CollectionConverters._
+
+class ApiClient {
```

Comment:
I've copied this pretty much directly from https://github.com/nationalarchives/prototype-scala-topic-listeners. If we stick with GraphQL, we should probably extract a TDR API client library.

Reply:
Agree.